modules/installation-azure-trusted-launch.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-network-customizations
+
+:_content-type: PROCEDURE
+[id="installation-azure-trusted-launch_{context}"]
+= Enabling trusted launch for Azure VMs
+
+You can enable two of Microsoft Azure's trusted launch features when installing your cluster: link:https://learn.microsoft.com/en-us/azure/virtual-machines/trusted-launch#secure-boot[secure boot] and link:https://learn.microsoft.com/en-us/windows/security/hardware-security/tpm/trusted-platform-module-overview[virtualized Trusted Platform Modules]. 
```

Comment:
Same comment about trying to rewrite to avoid making a company name possessive. I think it's okay sometimes, but I can't remember exactly when it's okay and when to avoid. So usually best to avoid. Here you could say "two of the Microsoft Azure trusted launch features"

I won't point out any more of these, if you can just double check throughout if you make updates

Reply:
Thanks, I fixed the rest of these.